pages/services/spark/2.3.1-2.2.1-2/hdfs/index.md

```diff
@@ -24,41 +27,58 @@ Within the {{ model.techShortName }} service configuration, set `hdfs.config-url
   }
 }
 ```
-This can also be done through the web interface. If you are using the default installation of HDFS from Mesosphere, this is probably `http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints`.
+This can also be done through the web interface. If you are using the default installation of HDFS from Mesosphere, this is probably http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints.
```

Comment:
why remove the backticks? intention is to have the code show as `as this` rather than a clickable url

Reply:
Whoops. Fixing....